epics-update.py

```diff
@@ -29,7 +29,7 @@
 
 from repo_defaults import *
 
-def update_pkg_dep_file( filePath, oldMacroVersions, newMacroVersions, verbose=False ):
+def update_pkg_dep_file( filePath, oldMacroVersions, newMacroVersions, options, verbose=False ):
```

Comment:
imo, it's better to have individual named arguments than to pass around an option parser.

Reply:
I tend to agree, but I have mixed feelings on this one. The number of arguments for the function is getting high and passing options make it more compact. At the same, time we lose clarity when we look at the function definition.

Reply:
I made this change when looking at this call and realizing we may want to add additional options in the future:
`        count += update_pkg_dependency( options.top, options.packages, options.noMY_MODULES, options.verbose )
`

Reply:
Another way to handle this would be to make noMY_MODULES a global variable.    Let me know if you prefer this approach.

Reply:
The original approach is okay then :)

Approved